spinn_front_end_common/utilities/report_functions/router_collision_potential_report.py

```diff
@@ -16,23 +16,17 @@
 import os
 from collections import defaultdict
 from spinn_machine.router import Router
-from spinn_front_end_common.utilities.globals_variables import (
-    report_default_directory)
+from spinn_front_end_common.data import FecDataView
 
 COLLISION_REPORT = "routing_collision_protential_report.rpt"
```

Comment:
while I'm in here, can we rename this report to change "protential" to "potential"?

Reply:
https://github.com/SpiNNakerManchester/SpiNNFrontEndCommon/issues/891